Merge opencog -> singnet

vsbogd opened the pull request.

linas committed:
Merge pull request #25 from singnet/master

Pass parses filename as parameter

linas committed:
Corrected description of pair data.

linas committed:
Add details from Andres.

Also print a new PDF; the old one is stale.

linas committed:
Merge pull request #27 from linas/ongoing

Update to diary PDF

vsbogd commented:
@glicerico , could you please take a look and merge?

glicerico commented:
I have no objections, all changes are in Linas' journal files... merging now

akolonin commented:
@glicerico - I have mentioned there was circleci test failure in the PR, does it matter?

glicerico commented:
@akolonin There have been failures on circleci for a while now, and I thought @vsbogd was creating these merges partly to try to solve that

vsbogd commented:
@akolonin, @glicerico,

The build on PR was red because I raised many PRs simultaneously and some of them have incompatible changes. I checked that they are compatible between each other using integration job which can check changes from set of branches and repos (I made a post about this in `opencog-fork` slack channel). Unfortunately I didn't add `learn` repo in this job from the start as this repo is not included into `cogutil` job which I used as template. I am sorry about that. Right now I have added `learn` component`. 

I checked results of the build after merge and I see that all unit test fails (see https://circleci.com/gh/singnet/learn/21). I see that this PR doesn't have functional changes at all, so it is not a reason and my understanding that it was broken before. Looks like next step is fix broken tests, as I said I already fixed integration job to prevent such situations in future (new integration job status is at https://circleci.com/workflow-run/b8788688-94d9-4242-8e92-eb6f5f6f3f61).
